### PR TITLE
Add CI config for Intly branch 3scale-next-0.7.0

### PR DESCRIPTION
**Pull request by psturc**
Based on a new 0.7.0 branch: https://github.com/integr8ly/integreatly-operator/pull/1249/files

### CODE REVIEW

**Comment by psturc** [comment]
cc @mikenairn

File: core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_3scale-next-0_7_0 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/integr8ly/3scale-next-0.7.0:integreatly-operator quay.io/integreatly/integreatly-operator:3scale-next-0.7.0
+registry.svc.ci.openshift.org/integr8ly/3scale-next-0.7.0:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:3scale-next-0.7.0
```
Comment (mikenairn) [inline core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_3scale-next-0_7_0:2]: You will need to add a new line here for the rhoam test harness image https://github.com/openshift/release/blob/master/core-services/image-mirroring/integr8ly/mapping_integr8ly_operator#L3. The delorean command you used to generate this should be updated as well ideally.

```registry.svc.ci.openshift.org/integr8ly/i3scale-next-0.7.0:integreatly-operator-test-harness-osde2e quay.io/integreatly/integreatly-operator-test-harness:osde2e-3scale-next-0.7.0 ```

**Comment by mikenairn** [comment]
/lgtm
/approve

@psturc You will need one of these to approve also https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/OWNERS#L4

File: core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_3scale-next-0_7_0 (hunk 1 above)
Comment (psturc) [reply core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_3scale-next-0_7_0:2]: update: eventually we will "merge" osde2e and functional test images into a single testing image that can be used for testing using osde2e framework or nightly jenkins pipelines

**Comment by laurafitzgerald** [comment]
/approve
failures in the tests are fine. 
manifest is failing because this is `-next` branch so the mapping is there. That one is relevant for merging so it's fine as it would be required to remove the mapping before the branch itself is merged.

**Pull request merged**